AdyenTests/Card Tests/CardComponentTests.swift

```diff
@@ -517,7 +517,7 @@ class CardComponentTests: XCTestCase {
                                              holderName: "holderName")
         let sut = CardComponent(paymentMethod: method,
                                 clientKey: "test_client_key")
-        sut.showsSecurityCodeField = false
```

Comment:
so the field `sut.showsSecurityCodeField ` was actually useless in this test?

Reply:
no, it wasn't, it was affecting both form and stored card pop at the same time, I made them separate.